exercism-io.md

```diff
@@ -0,0 +1,163 @@
+# exercism.io - Javascript
+
+## Overview
+
+Exercism.io is a set of mini-quests designed to introduce you to a new language, in this case Javascript.  It will present you with problems and a set of test suites to verify that your solution is correct.  Learning by TDD!  It's a great way to explore the language.
+
+## Setup
+
+To install Exercism first install Nodejs (you may have this already installed.
+
+```bash
+$  brew install node
```

Comment:
Will do, although I'll leave it commented in the doc in case we want them installing this on their own before JS starts next time.